### DISCUSSION TITLE
Limit thumbnail images to 1

### DISCUSSION THREAD

**Discussion by lalucacreative** [fancyapps/ui#212] [Q&A]
So glad I discovered the Fancybox, thanks for developing and sharing this! Is it possible to only display 1 image thumbnail on the page, and when the user clicks the image it opens the lightbox with more images? Thanks!

**Comment by thewebartisan7** [comment]
Check this example https://fancyapps.com/docs/ui/fancybox/plugins/image#gallery-with-one-preview-image
